<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Qt_TCPServerClass</class>
 <widget class="QWidget" name="Qt_TCPServerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qt_TCPServer</string>
  </property>
  <widget class="QGroupBox" name="receiveBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>361</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>接收窗口</string>
   </property>
   <widget class="QTextEdit" name="recvEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>341</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="sendBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>581</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>发送窗口</string>
   </property>
   <widget class="QPushButton" name="sendBtn">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>20</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="sendEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>16</y>
      <width>471</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>201</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>网络设置</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>20</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1)本机IP地址</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="IPEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>2)本地端口号</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="portNumEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="connectBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>181</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>连接</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>hex显示</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearBtn">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>清除窗口</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>全部连接</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Qt_TCPServer.qrc"/>
 </resources>
 <connections/>
</ui>
